docs/admin/cluster-large.md

```diff
@@ -75,6 +75,10 @@ However, this is done only for clusters having more than 50 nodes.
 
 ### Addon Resources
 
+** TODO: Update this to reflect
```

Comment:
This didn't make the TODO bold, maybe because there are spaces between before and after the **